lib/diff.js

```diff
@@ -245,6 +250,27 @@ function diffLayers(before, after, commands) {
     }
 }
 
+function diffLight(before, after, commands) {
+    before = before || {};
+    after = after || {};
+
+    var lightProps = ['anchor', 'direction', 'color', 'intensity'];
+
+    var newProps = {};
+
+    for (var i in lightProps) {
```

Comment:
Creating a minimal diff feels like overkill. We can check if `!isEqual(before, after)` and push a command for the entire `after` lighting object if so.

Iterating over each property was only necessary when the `setLighting` command accepted a single property per invocation.